docs/source/user/parallel.rst

```diff
@@ -95,6 +96,21 @@ reduction (``A`` is a one-dimensional Numpy array)::
             s += A[i]
         return s
 
+The following example demonstrates a product reduction on a two-dimensional array::
+
+    from numba import njit, prange
+    @njit(parallel=True)
+    def two_d_array_reduction_prod(n):
+        shp = (13, 17)
+        size = shp[0] * shp[1]
```

Comment:
Just realised, `size` is dead. It's also dead in the test case that this is based on.

Reply:
Done.

Reply:
thanks :)